### DISCUSSION TITLE
无法登录

### DISCUSSION THREAD

**Discussion by foredawn3** [cssbattle/forum#259] [Q&A]
使用邮箱登录显示把无密码登录链接发送到了邮箱，但是邮箱没收到
![image](https://user-images.githubusercontent.com/96131200/167412743-fba3e759-0464-4a6e-b8f5-e7cc81f1d17d.png)

**Comment by chinchang** [comment]
PLease check spam folder

**Comment by Guagua111-OIer** [comment]
我也是
